File: help/pt_pt/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_007 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Teorema de Pitágoras\n" );
        System.out.print("Cateto a \t: " );
        double a = Double.valueOf(keyb.nextLine());
        System.out.print("Cateto b \t: " );
        double b = Double.valueOf(keyb.nextLine());
        double c = Math.sqrt( Math.pow( a , 2 ) + Math.pow( b , 2 ) ) ;
        System.out.print("Hipotnusa \t: " + c );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:29

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Pythagorean Theorem\n" ;
    echo  "Catet a     : " ;
    $a = floatval(isset($_POST["a"])? $_POST["a"] : isset($_GET["a"]) ? $_GET["a"] : 0);
    
    echo $a . "<br/>";// print input 
    echo  "Catet b     : " ;
    $b = floatval(isset($_POST["b"])? $_POST["b"] : isset($_GET["b"]) ? $_GET["b"] : 0);
    
    echo $b . "<br/>";// print input 
    $c = sqrt( pow( $a , 2 ) + $b ^ 2 ) ;
    echo  "Hypotnusa     : " . $c  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:56

?>